misk/src/test/kotlin/misk/web/extractors/FormValueParameterTest.kt

```diff
@@ -0,0 +1,169 @@
+package misk.web.extractors
+
+import com.google.inject.util.Modules
+import com.squareup.moshi.Moshi
+import misk.MiskModule
+import misk.inject.KAbstractModule
+import misk.testing.MiskTest
+import misk.testing.MiskTestModule
+import misk.testing.TestWebModule
+import misk.web.*
+import misk.web.actions.WebAction
+import misk.web.jetty.JettyService
+import misk.web.mediatype.MediaTypes
+import okhttp3.FormBody
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import javax.inject.Inject
+
+@MiskTest(startService = true)
+internal class FormValueParameterTest {
+    data class Packet(val message: String)
```

Comment:
not needed?

Reply:
yep! copied from somewhere else. Removed from both